### ISSUE TITLE
tx_bootstrappackage_card_group_item can not uncheck visibility "hidden" after setting card item to "hidden"

### ISSUE THREAD

**Issue by MarcoHaase** [benjaminkott/bootstrap_package#1205]
# Bug Report

If you set visibility to "hidden" in "Access" tab on a "tx_bootstrappackage_card_group_item", you can not uncheck it by the "hide record" controller icon right top, so card item is still hidden without error message.

## Prerequisites

* [ Yes ] Can you reproduce the problem on TYPO3 v10.4 LTS
* [ Yes ] Can you reproduce the problem on TYPO3 v11.3
* [ Yes ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Abenjaminkott+repo%3Abenjaminkott%2Fbootstrap_package)
  to see if your bug or enhancement is already reported?

## Description

If you set visibility to "hidden" in "Access" tab on a "tx_bootstrappackage_card_group_item", you can not uncheck it by the "hide record" controller icon right top, so card item is still hidden without error message.

If you only use "hide record" icon to hide and unhide item, it works fine, but if you set 1 time visibility checkbox, it is broken.

there is an error using:

```
                'items' => [
                    '1' => [
                        '0' => 'LLL:EXT:frontend/Resources/Private/Language/locallang_ttc.xlf:hidden.I.0'
                    ]
                ]
```

seems to be ` '0' => [` instead of ` '1' => [` or missing ...['items'][0] element.

Workarround to fix issue:

../Configuration/TCA/Overrides/100_bootstrappackage_card_group_item.php:

```
defined('TYPO3') or die('Access denied.');

// Activate T3EDITOR if extension is activated
if (\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::isLoaded('t3editor')) {
    $GLOBALS['TCA']['tx_bootstrappackage_card_group_item']['columns']['hidden']['config']['items'] = [
        ['LLL:EXT:frontend/Resources/Private/Language/locallang_ttc.xlf:hidden.I.0'],
    ];
}
```

## Steps to Reproduce

1. Create card_group_item
2. check "Disable" checkbox at "Access" tab at "Visibility"
3. save content element
4. use checkbox or "hide record" icon
5. broken

### Expected behavior

card item unhide

### Actual behavior

card item still hide

### Screenshots

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/86526555/194510141-d212ab55-5067-4b2f-8599-5d3f6f41efd3.png">

checkbox:
<img width="128" alt="image" src="https://user-images.githubusercontent.com/86526555/194510205-edfee03b-bd3e-40cd-ad0e-c4e5fbf03b0f.png">

hide record icon:
<img width="49" alt="image" src="https://user-images.githubusercontent.com/86526555/194510229-97d50aaf-3e2d-4c49-98ea-9390d19787e2.png">

## Versions

You can get this information from executing `composer show` at the command line
if your TYPO3 installation is composer based, otherwise the version number will
be shown in the extension manager.

**Comment by mirkanglanz** [comment]
I have a similar issue probably related to this, with TYPO3 12.2, bootstrap_package 13.0.1 without having used the "Visibility - Card - Disable checkbox".

When I accordion-open a card within a card group using the little arrow button, I can not unhide it with the "unhide_record" switch. 
It only works if the card is not accordion-opened.

If you just unhide without having it opened, it works. It works on the 2nd try if you just closed the card.

![boostrap_cards_issue_visibility](https://user-images.githubusercontent.com/95470556/223682443-e23a614f-167d-4640-a974-7097e4bdeae0.png)

**Comment by tbsschmdt** [comment]
This is fixed in [Version 13.0.2](https://github.com/benjaminkott/bootstrap_package/releases/tag/13.0.2) via Pull request #1247.
